src/shortcuts/bold.ts

```diff
@@ -17,11 +15,8 @@ const bold: Shortcut = {
   exec: dispatch => {
     dispatch(formatSelection('bold'))
   },
-  isActive: getState => {
-    const state = getState()
-    if (!state.cursor) return false
-    const thought = getThoughtById(state, head(state.cursor))
-    return thought.value.includes('<b>') || thought.value.includes('<strong>')
+  isActive: () => {
+    return false
   },
```

Comment:
The fact that the active state results from a combination of the `return false` here and the logic in `ToolbarButton` is nonobvious. Instead of splitting the logic, we should either have it all in `isActive`, or all in `ToolbarButton`. 

For simplicity I think we should have the active logic for the formatting commands determined wholly in `ToolbarButton`. If the toolbar button is associated with one of the command states, it should ignore `shortcut.isActive`. The isActive definitions can be removed and replaced with a comment that developers to the `ToolbarButton` logic.

`isActive` is a Redux selector and would require a significant refactor to support non-Redux stores. #2213 is not quite right, as [noted](https://github.com/cybersemics/em/pull/2213#discussion_r1702225865).

Reply:
I like that suggestion. I've implemented it here: ace1fc0508b500354405a2354f4b53b417910970